README.md

```diff
@@ -1,33 +1,41 @@
 # Zoom Recordings Downloader
 
+> This [README in Spanish](README.es.md)
+
 A Python library to download Zoom users recordings
 
-## Dependencias
+## Arquitecture and requirements
+
+To automatically access recordings for download, you must have a Zoom account with developer privilegies in Zoom platform. You can then dowonload the recordings of all the virtual rooms that your account has aqcces to.
+
+The library is called from a program running in server or a local system with Python, and accesses Zoom platform ressources through an [API JWT](https://marketplace.zoom.us/docs/guides/auth/jwt).
+
+## Dependencies on your system
 
-* Python 3,6+
-* Además de instalar los siguientes paquetes requeridos:
+* Python version 3.6 or higher
+* Morover, you must install the following packages:
    * zoomus
    * python-decouple
    * python-dateutil
 
-En sistemas Debian puede instalar las dependencias necesarias así:
+On Debian and debian-like systems, you can install the needed dependencies as follows: 
 ```
-`apt install python3-venv python3-pip`
-`pip3 install zoomus`
-`pip3 install python-decouple`
-`pip3 install python-dateutil`
+apt install python3-venv python3-pip
+pip3 install zoomus`
+pip3 install python-decouple
+pip3 install python-dateutil
 ```
 
-## Uso
+## Usage
 
-Primero deberá crear una JWT App accediendo con las credenciales de su cuenta en el [Marketplace de Zoom](https://marketplace.zoom.us/) para lo cual se recomiendoa seguir [esta documentación](https://marketplace.zoom.us/docs/guides/build/jwt-app).
+First you must create a JWT App, accessing with your credentials inti the [Zoom Marketplace](https://marketplace.zoom.us/), following this [this documentation](https://marketplace.zoom.us/docs/guides/build/jwt-app).
```

Comment:
typo: "inti"

Reply:
"Int"` means "sun" in Quechua. The week end was cold...